src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
     <h1>Moyo header</h1>
```

Comment:
Think this header `<h1>Moyo header</h1>`, regarding the designs should be removed

![Screenshot 2023-07-04 at 07 44 21](https://github.com/mate-academy/layout_moyo-header/assets/30219274/42c418a0-355a-4d3d-9f66-2171fa80078e)